themes/default/theme/src/scss/docs/_docs-main.scss

```diff
@@ -90,6 +90,7 @@ body.section-registry {
             padding: 0 0 12px 0;
             margin: 0;
             line-height: 1.6;
+            overflow-wrap: break-word;
```

Comment:
Should we scope this to tables? Only concerned we might end up with broken words where we aren't currently expecting them. (Clicking around so far, though, this looks okay.)

Reply:
yeah probably not a bad idea